nut/DOCS.md

```diff
@@ -35,7 +35,8 @@ comparison to installing any other Home Assistant add-on.
 1. Configure the `users` and `devices` options.
 1. Start the "Network UPS Tools" add-on.
 1. Check the logs of the "Network UPS Tools" add-on to see if everything went well.
-1. Configure the [NUT Integration][nut-ha-docs].
+1. Configure the [NUT Integration][nut-ha-docs] using Host `a0d7b954-nut`,
+   Port `3493`, and the Username/Password configured in the add-on.
```

Comment:
This is an unrelated change to the PR description.
Additionally, this is not always true. Depending on how you install it, the host might differ; thus documenting this as such is not correct.

Let's revert this change.

Reply:
Done.  I'll open a separate PR, as I think further discussion is warranted, although there is no need to tie it to this change.